common/components/chrome/SubHeader.jsx

```diff
@@ -43,26 +61,228 @@ class SubHeader extends React.Component<{||}, State> {
+                  </a>
+                </ClickAwayListener>
+              </Paper>
+            </Grow>
+          )}
+        </Popper>
+      </div>
+    )
+  }
+
+  _toggleDropdown(): void {
+    this.setState({ dropdown: !this.state.dropdown });
+  };
+
+  _closeDropdown(event): void {
+    if (this.anchorEl.contains(event.target)) {
+      return;
+    };
+    this.setState({ dropdown: false });
+  };
+
+  _renderHamburgerSlider(): React$Node {
+    return (
+      <React.Fragment>
+        <div className="SubHeader-hamburger" onClick={() => this._toggleSlider(true)}>
+          <IconButton >
+            <MenuIcon />
+          </IconButton>
+        </div>
+        <Drawer open={this.state.slider} onClose={() => this._toggleSlider(false)} styles={{overflow: 'hidden'}}>
+          <div
+            tabIndex={0}
+            role="button"
+            onClick={() => this._toggleSlider(false)}
+            onKeyDown={() => this._toggleSlider(false)}
+          >
+            <div className="SubHeader-navDrawer">
+            {
+              CurrentUser.isLoggedIn() &&
+              <div>
+                <div className='SubHeader-slider-icon'>
+                  {this._renderIcon()}
+                </div>
+                <p className='SubHeader-slider-name'>{CurrentUser.firstName() + ' ' + CurrentUser.lastName()} </p>
+                <a href="" onClick={() => this._navigateToSection('MyProfile')}>
+                  <div className={'SubHeader-drawerDiv'} >
+                    My Profile
+                  </div>
+                </a>
+                <Divider />
+                <a href="" onClick={() => this._navigateToSection('MyProjects')}>
+                  <div className={'SubHeader-drawerDiv'} >
+                    My Projects
+                  </div>
+                </a>
+                <Divider />
+              </div>
+              
+            }
+              <a href="" onClick={() => this._navigateToSection('FindProjects')}>
+                <div className={'SubHeader-drawerDiv'} >
+                  Find Projects
+                </div>
+              </a>
+              <Divider />
+              <a href="http://connect.democracylab.org" target="_blank" rel="noopener noreferrer">
+                <div className={'SubHeader-drawerDiv'} onClick={() => this._navigateToSection('FindProjects')}>
+                  About
+                </div>
+              </a>
+              <Divider />
+              <a href="mailto:hello@democracylab.org" target="_blank" rel="noopener noreferrer">
+                <div className={'SubHeader-drawerDiv'} onClick={() => this._navigateToSection('FindProjects')}>
+                  Contact Us
+                </div>
+              </a>
+              <Divider />
+              {
+                !CurrentUser.isLoggedIn() &&
+                <React.Fragment>
+                  <div className={'SubHeader-drawerDiv'} onClick={() => this._navigateToSection('LogIn')}>
```

Comment:
There's a `this._onLogInClick` function used for the not-mobile login button, is there a reason not to reuse it here?